code/datums/components/supermatter_crystal.dm

```diff
@@ -288,15 +288,23 @@
 	consume(atom_source, nom)
 
 /datum/component/supermatter_crystal/proc/consume(atom/source, atom/movable/consumed_object)
+	if((consumed_object.flags_1 & SUPERMATTER_IGNORES_1))
```

Comment:
```suggestion
	if(consumed_object.flags_1 & SUPERMATTER_IGNORES_1)
```

Reply:
you just know some day this will be changed, they wont include parenthesis, and the check will break

Reply:
Would be wise to have a unit test then